torch/quantization/observer.py

```diff
@@ -35,27 +35,29 @@ def forward(self, x):
         else:
             self.min_val = torch.min(torch.min(x), self.min_val)
             self.max_val = torch.max(torch.max(x), self.max_val)
+        return x
```

Comment:
Although it's OK for this time, generally we would want to separate unrelated fixes. This fix is part of a different diff if I remember correctly -- in the future you can just stack diffs